### PR TITLE
Add MacOS installation guide

### PR DESCRIPTION
**Pull request by TheNinty**
Adds an installation guide for MacOS, and splits up the Windows and MacOS guides.

### CODE REVIEW

File: wiki/Installation/en.md [hunk 1]
```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
 5.  osu! will prompt you to sign in or register.
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
+
+- Since the MacOS version of osu! is unofficial, the game may not fuction properly, or even boot at all.
```
Comment (MegaApplePi) [inline wiki/Installation/en.md]:
```suggestion
The MacOS version of osu! is experimental. The game may not function properly or even boot at all. 
```

File: wiki/Installation/en.md [hunk 2]
```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
...
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
+
+- Since the MacOS version of osu! is unofficial, the game may not fuction properly, or even boot at all. 
+
+1.  Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
+2.  Locate the installer, right click, and select open.
+    - Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified devloper. Right clicking and selecting open will solve this problem.
```
Comment (MegaApplePi) [inline wiki/Installation/en.md]:
```suggestion
    -   Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified developer. Right clicking and selecting open will solve this problem.
```

File: wiki/Installation/en.md [hunk 3]
```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
...
+3.  The installer will automatically run. You cannot change the installation location, since it is installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.
```
Comment (MegaApplePi) [inline wiki/Installation/en.md]:
```suggestion
3.  The installer will automatically run. The installation location cannot be changed. It will be installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.
```

File: wiki/Installation/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Installation/en.md]: was the space at the front intentional

Comment (MegaApplePi) [reply wiki/Installation/en.md]: Yes, it would keep the spacing consistent with the numbered list.

**Comment by TheNinty** [comment]
Should there be separate articles for each installation method?

**Comment by TPGPL** [comment]
@TheNinty you could also delete L3 and replace `This page will roughly tell you how to get osu! working on your Windows device.` with `This page will roughly tell you how to get osu! working on your various devices.`

**Comment by TheNinty** [comment]
Some of the methods for doing other things are different, like adding a skin.

File: wiki/Installation/en.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 <!-- *See also: [Installation/OS X](/wiki/Installation/OS_X) and [Installation/Linux](/wiki/Installation/Linux)* -->
 
-This page will roughly tell you how to get osu! working on your Windows device. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.
+This page will roughly tell you how to get osu! working on various devices. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.
```
Comment (TPGPL) [inline wiki/Installation/en.md]: revert this change

File: wiki/Installation/en.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 <!-- *See also: [Installation/OS X](/wiki/Installation/OS_X) and [Installation/Linux](/wiki/Installation/Linux)* -->
```
Comment (TPGPL) [inline wiki/Installation/en.md]:
```suggestion
*See also: [Installation/Mac_OS](/wiki/Installation/Mac_OS)* <!-- and [Installation/Linux](/wiki/Installation/Linux)* -->
```

File: wiki/Installation/en.md [hunk 6]
```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
 5.  osu! will prompt you to sign in or register.
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
```
Comment (TPGPL) [inline wiki/Installation/en.md]: move this section to a `wiki/Installation/Mac_OS/en.md article 

https://gist.github.com/TPGPL/1beea5d9293dfe5e5781f8e29060d2e4 - article's content

File: wiki/Installation/en.md (hunk 5 above)
Comment (MegaApplePi) [reply wiki/Installation/en.md]: I think it is actually spelt as `macOS` with lowercase M and no space.

**Comment by TheNinty** [comment]
ok I made a new article at `wiki/Installation/macOS/en.md` and reverted the old one to it's original state

File: wiki/Installation/en.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 <!-- *See also: [Installation/OS X](/wiki/Installation/OS_X) and [Installation/Linux](/wiki/Installation/Linux)* -->
 
-This page will roughly tell you how to get osu! working on your Windows device. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.
+This page will roughly tell you how to get osu! working on your windows device. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ) that explains how to install osu! on Windows.
```
Comment (TPGPL) [inline wiki/Installation/en.md]: capitalize Windows

File: wiki/Installation/en.md (hunk 5 above)
Comment (TPGPL) [reply wiki/Installation/en.md]: hasn't been addressed

**Comment by TheNinty** [comment]
the installation en.md file now has a link to the macOS version of the installer
